packages/api/test/scripts/constants.js

```diff
@@ -1,3 +1,3 @@
 export const endpoint = 'http://localhost:8787'
-export const clusterApi = 'http://localhost:9094'
+export const clusterApi = 'http://127.0.0.1:9094'
```

Comment:
Seems that it's because I'm using Node.js 18...which resolves localhost to `::1` on MacOS (no longer reorders for consistent resolution across platform).